nmdc_runtime/api/endpoints/jobs.py

```diff
@@ -2,14 +2,16 @@
 from typing import Optional, Annotated
 
 import pymongo
-from fastapi import APIRouter, Depends, Query
+from fastapi import APIRouter, Depends, Query, HTTPException
```

Comment:
```suggestion
from fastapi import APIRouter, Depends, Query, HTTPException, Path
```

Added an import to accommodate [the suggestion below](https://github.com/microbiomedata/nmdc-runtime/pull/962/files#r2051136031).